networkit/cpp/edgescores/test/EdgeScoreAsWeightGTest.cpp

```diff
@@ -0,0 +1,81 @@
+/*
+ * EdgeScoreGTest.cpp
+ *
+ *	Created on: 16.10.2025
+ *  Authors: Andreas Scharf (andreas.b.scharf@gmail.com)
+ */
+#include <gtest/gtest.h>
+
+#include <networkit/edgescores/EdgeScoreAsWeight.hpp>
+#include <networkit/graph/Graph.hpp>
+#include <networkit/graph/GraphTools.hpp>
+
+namespace NetworKit {
+
+static std::vector<double> makeSequentialScores(const Graph &G) {
+    std::vector<double> s(G.numberOfEdges());
+    for (edgeid eid = 0; eid < G.numberOfEdges(); ++eid) {
+        s[eid] = static_cast<double>(eid + 1); // 1.0, 2.0, 3.0, ...
+    }
+    return s;
+}
+
+TEST(EdgeScoreAsWeight, testThrowsIfEdgesNotIndexed) {
```

Comment:
The test name should be `EdgeScoreAsWeightGTest` (as well for the other occasions).